_sass/custom/_accordion.scss

```diff
@@ -28,9 +29,8 @@
         }
 
         .accordion-icon {
-            position: absolute;
-            top: 0;
-            right: 0;
+            position: relative; 
+            margin-left: auto; /* move icon to the right */
```

Comment:
Let's remove the comment here since we do not necessarily need it @jpmaHub

Reply:
@khairahscorner I have removed the comment and the bug is only replicate from safari browser